R/model-modules.R

```diff
@@ -10,7 +10,7 @@
 #'
 #' @param distribution A character vector describing the parametric delay
 #' distribution to use. Current options are: "none", "lognormal", "gamma",
-#' and "exponential" with the default being "lognormal".
+#' "exponential", and "loglogistic", with the default being "lognormal".
```

Comment:
should we link out to distribution definitions here do you think? I particularly wonder if the different definitions of summary parameters will confuse people. Perhaps this is something for another issue.

Reply:
Yes, I agree, will make a new issue for this. Definitely important since they also translate differently into the moments of the distributions.